tests/sentry/api/endpoints/test_auth_index.py

```diff
@@ -117,10 +118,13 @@ def test_valid_password_u2f(self, validate_response, is_available):
         assert {"response": "response"} in validate_response.call_args[0]
 
 
-class AuthVerifyEndpointSuperuserTest(AuthProviderTestCase, APITestCase):
+class AuthVerifyEndpointSuperuserTest(AuthProviderTestCase, AuthVerifyEndpointTest, APITestCase):
```

Comment:
This is going to run the tests in `AuthVerifyEndpointTest` twice. Grab the functions you need and put it in a MixIn instead.